### DISCUSSION TITLE
Codium not launching as Wayland app --- workaround

### DISCUSSION THREAD

**Discussion by JPDucky** [VSCodium/vscodium#1098] [General]
If you are having issues with scaling, chances are it's an electron bug and is launching in Xwayland, but there is a quick and easy workaround that I found [here](https://github.com/microsoft/vscode/issues/146349). Just launch it with ```codium --enable-features=UseOzonePlatform --ozone-platform=wayland --log debug``` and for some reason adding the ```--log debug``` makes it listen to the previous two commands. I have not been able to get Codium to launch as a wayland app while ommitting the debug part. 

***I didn't want to create an issue as this is an electron bug not codium, but I did want to put this workaround somewhere other people with this issue can see it

**Comment by paulcarroty** [comment]
Thx for the tip, sadly failed on 1.68.

**Reply by JPDucky** [reply]
try adding additional debug tags to the end of the command ie `--log debug --log debug`. I saw that had worked for someone, it took as many as 5 for another
